ajax/libs/jquery-nstslider/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "jquery-nstslider",
+  "filename": "jquery.nstSlider.min.js",
+  "description": "Fully customizable with CSS, Single/Double handles, Touch-enabled, IE 7+ Compatibility, Custom Digit Rounding, Non linear step increments!",
+  "keywords": [
+    "Single/Double handles",
+    "Touch-enabled",
+    "IE 7+ Compatibility",
+    "Custom Digit Rounding"
+  ],
```

Comment:
These are not "keywords", you only used a complex string, think about what does keywords mean, and see: https://github.com/lokku/jquery-nstslider/blob/master/nstSlider.jquery.json